windows/dockerfile-to-powershell-generated-sample.ps1

```diff
@@ -97,113 +97,146 @@ function Update-SessionEnvironment {
 
-# Line: 49, Content: LABEL windows_version=${WINDOWS_VERSION}
+# Line: 61, Content: LABEL windows_version=${WINDOWS_VERSION}
 # LABEL resolved: windows_version='1809'
 
-# Line: 50, Content: LABEL git_version=${GIT_VERSION}
+# Line: 62, Content: LABEL git_version=${GIT_VERSION}
 # LABEL resolved: git_version='2.26.2'
 
-# Line: 51, Content: LABEL sevenzip_version=${SEVENZIP_VERSION}
+# Line: 63, Content: LABEL sevenzip_version=${SEVENZIP_VERSION}
 # LABEL resolved: sevenzip_version='19.0.0'
 
-# Line: 52, Content: LABEL vs2017buildtools_version=${VS2017BUILDTOOLS_VERSION}
+# Line: 64, Content: LABEL vs2017buildtools_version=${VS2017BUILDTOOLS_VERSION}
 # LABEL resolved: vs2017buildtools_version='16.8.3.0'
 
-# Line: 53, Content: LABEL vcpython27_version=${VCPYTHON27_VERSION}
-# LABEL resolved: vcpython27_version='9.0.0.30729'
-
-# Line: 54, Content: LABEL go_version=${GO_VERSION}
+# Line: 65, Content: LABEL go_version=${GO_VERSION}
 # LABEL resolved: go_version='1.17.6'
 
-# Line: 55, Content: LABEL ruby_version=${RUBY_VERSION}
+# Line: 66, Content: LABEL ruby_version=${RUBY_VERSION}
 # LABEL resolved: ruby_version='2.6.6-1'
 
-# Line: 56, Content: LABEL wix_version=${WIX_VERSION}
+# Line: 67, Content: LABEL wix_version=${WIX_VERSION}
 # LABEL resolved: wix_version='3.11.2'
 
-# Line: 57, Content: LABEL cmake_version=${CMAKE_VERSION}
-# LABEL resolved: cmake_version='3.17.2'
+# Line: 68, Content: LABEL cmake_version=${CMAKE_VERSION}
+# LABEL resolved: cmake_version='3.23.0'
 
-# Line: 58, Content: LABEL msys_version=${MSYS_VERSION}
+# Line: 69, Content: LABEL msys_version=${MSYS_VERSION}
 # LABEL resolved: msys_version='20210725'
 
-# Line: 59, Content: LABEL system_python_version=${PYTHON_VERSION}
+# Line: 70, Content: LABEL system_python_version=${PYTHON_VERSION}
 # LABEL resolved: system_python_version='3.8.2'
 
-# Line: 60, Content: LABEL embedded_py2_version=${EMBEDDED_PYTHON_2_VERSION}
+# Line: 71, Content: LABEL embedded_py2_version=${EMBEDDED_PYTHON_2_VERSION}
 # LABEL resolved: embedded_py2_version='2.7.17'
 
-# Line: 61, Content: LABEL embedded_py3_version=${EMBEDDED_PYTHON_3_VERSION}
+# Line: 72, Content: LABEL embedded_py3_version=${EMBEDDED_PYTHON_3_VERSION}
 # LABEL resolved: embedded_py3_version='3.8.1'
 
-# Line: 62, Content: LABEL nuget_version=${NUGET_VERSION}
+# Line: 73, Content: LABEL nuget_version=${NUGET_VERSION}
 # LABEL resolved: nuget_version='5.8.0'
 
 # Add certificates needed for build & check certificates file hash
 # We need to trust the DigiCert High Assurance EV Root CA certificate, which signs python.org,
 # to be able to download some Python components during the Agent build.
-# Line: 66, Content: RUN (New-Object System.Net.WebClient).DownloadFile(\"https://curl.haxx.se/ca/cacert-${ENV:CACERTS_VERSION}.pem\", \"cacert.pem\")
+# Line: 77, Content: RUN (New-Object System.Net.WebClient).DownloadFile(\"https://curl.haxx.se/ca/cacert-${ENV:CACERTS_VERSION}.pem\", \"cacert.pem\")

```

Comment:
Technically code-review here is a formality because the file is fully generated. Of course you are right but you it helps a lot to pin-point a problem and trace it back to the docker file.